extensions/http/jersey/src/main/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/EdcApiExceptionMapper.java

```diff
@@ -66,8 +67,11 @@ public Response toResponse(Throwable exception) {
 
         if (exception instanceof EdcException && verboseResponse) {
             var edcApiException = (EdcException) exception;
-            var responseBody = List.of(Map.of("message", edcApiException.getMessage()));
-            return Response.status(status).entity(responseBody).build();
+            var apiError = ApiErrorDetail.Builder.newInstance()
+                    .message(edcApiException.getMessage())
+                    .type(edcApiException.getClass().getSimpleName().replace("Exception", ""))
```

Comment:
this feels a bit brittle. How about we introduce an `EdcApiException` that contains a `type` field? Shouldn't be too much impact, as it would only touch the controllers.

Reply:
@paullatzelsperger fine with adding a `type` to the `EdcApiExtension`, but in the `EdcApiExceptionMapper` we are handling also an `EdcException`.
Probably the `EdcQueryException` would make more sense as an `Api` one, I'll work on that.

Reply:
we also handle some standard JDK exceptions, such as `IllegalArgumentException`, `UnsupportedOperationException`. 
What if the `exceptionMap` contains not just the `Response.Status`, such as `BAD_REQUEST`, but a tuple consisting of the status _and_ a type string?